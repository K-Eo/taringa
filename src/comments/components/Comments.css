.Comments .btn-light {
  background: transparent;
  border-color: transparent;
}
