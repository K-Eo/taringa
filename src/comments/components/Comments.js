import React, { Component } from 'react'

import infiniteScroll from '../../HOC/InfiniteScroll'
import Comment from './Comment'

export class Comments extends Component {
  renderCommentsLabel() {
    const { totalCount } = this.props

    if (totalCount === 0) {
      return 'No hay comentarios'
    } else if (totalCount === 1) {
      return '1 comentario'
    } else {
      return `${totalCount} comentarios`
    }
  }

  handleRetryLoadMore(e) {
    e.preventDefault()

    if (this.props.status !== 'fetching') {
      this.props.loadMore()
    }
  }

  render() {
    const { canRender, comments, status, totalCount } = this.props

    if (!canRender) {
      return null
    }

    const canRenderCommentsCount =
      'comments' in this.props && 'totalCount' in this.props
    const canRenderComments = comments && comments.length > 0
    const canRenderSpinner = status === 'fetching'
    const hasError = status === 'failure'
    const hasNoMoreComments =
      comments &&
      comments.length === totalCount &&
      comments.length !== 0 &&
      status === 'success'

    return (
      <div className="Comments mt-4">
        <div className="card">
          {canRenderCommentsCount && (
            <div className="card-body">
              <h6 className="card-title mb-0">{this.renderCommentsLabel()}</h6>
            </div>
          )}

          {canRenderComments && (
            <div className="card-body">
              {comments.map(i => <Comment key={i.id} {...i} />)}
            </div>
          )}

          {canRenderSpinner && (
            <div className="card-body">
              <div className="text-center">
                <i className="fa fa-circle-notch fa-spin" />
              </div>
            </div>
          )}

          {hasError && (
            <div className="text-center">
              <p className="text-danger">
                ¡Ratas! Algo salío mal. Refresca la página o haz clic{' '}
                <a href="/retry" onClick={this.handleRetryLoadMore.bind(this)}>
                  aquí
                </a>{' '}
                para volver a intentar.
              </p>
            </div>
          )}

          {hasNoMoreComments && (
            <div className="text-center">
              <p className="text-secondary">No hay más comentarios</p>
            </div>
          )}
        </div>
      </div>
    )
  }
}

Comments.defaultProps = {
  canRender: true,
  comments: [],
  loadMore: () => {},
  status: 'fetching',
}

export default infiniteScroll()(Comments)
