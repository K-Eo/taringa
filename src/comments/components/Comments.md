Comments idle

```js
const Comments = require('./Comments').Comments;

<Comments canRender={false} />
```

Comments empty

```js
const Comments = require('./Comments').Comments;

<Comments
  totalCount={0}
  status="success"
/>
```

Comments listing

```js
const comment = require('./comment.data').comment;
const Comments = require('./Comments').Comments;

<Comments
  totalCount={1}
  status="success"
  comments={[comment]}
/>
```

Comments listing more than one

```js
const comments = require('./comment.data').comments;
const Comments = require('./Comments').Comments;

<Comments
  totalCount={2}
  status="success"
  comments={comments}
/>
```

Comments listing with loading more

```js
const comment = require('./comment.data').comment;
const Comments = require('./Comments').Comments;

<Comments
  totalCount={1}
  status="fetching"
  comments={[comment]}
/>
```

Comments listing with error

```js
const comment = require('./comment.data').comment;
const Comments = require('./Comments').Comments;

<Comments
  totalCount={1}
  status="failure"
  comments={[comment]}
/>
```
