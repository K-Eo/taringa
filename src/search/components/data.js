export const user = {
  id: '27s73',
  created: 1513920502005,
  type: 'user',
  tid: 924744,
  username: 'supr159',
  firstname: 'Diego',
  lastname: 'Lopez',
  birthday: '1992-1-18',
  avatar:
    'https://a21.t26.net/taringa/avatares/4/6/6/F/D/5/supr159/120x120_B7B.jpg',
  site: '',
  gender: 'm',
  country: 'MX',
  message: '',
  subscribedToDefaultChannels: true,
  suspended: false,
  stories: 11,
  followers: 32,
  following: 0,
  rank: 'User',
  latestStory: '2018-01-17T03:07:26.000Z',
  state: {
    following: false,
  },
}
