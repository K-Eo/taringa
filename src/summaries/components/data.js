export const image = {
  id: 'zbmsb',
  created: 1513920502005,
  edited: null,
  title: '',
  nsfw: false,
  classic: {
    tid: 'cH3t9',
    type: 'shout',
  },
  tags: ['image', 'classic'],
  slug: 'si-no-das-like-no-tendras-pelirojas-en-100-anos_zbmsb',
  type: 'story:summary',
  summary: {
    link: null,
    images: {
      slice: [
        {
          type: 'image',
          url: 'https://k50.kn3.net/taringa/2/7/9/1/4/3/20/fraudier/D4C.jpg',
          width: 779,
          height: 1026,
          format: 'jpeg',
        },
      ],
      amount: 1,
    },
    excerpt: 'Si no das like no tendras pelirojas en 100 años',
    textLength: 47,
  },
  channel: {
    type: 'channel:summary',
    id: 'ixoir',
    created: '2016-08-28T21:54:19.000Z',
    name: 'user-Fraudier',
    title: 'user-Fraudier',
    channelType: 'user_feed',
    thumbnail: '',
    background: '',
    description: '',
    category: 'unknown/unknown',
    owner: {
      id: 'ixoiq',
      created: '2016-08-28T21:54:19.000Z',
      type: 'user',
      tid: 27914320,
      username: 'Fraudier',
      firstname: 'Fraudier',
      lastname: 'ㅤ',
      birthday: '1910-9-19',
      avatar:
        'https://a12.t26.net/taringa/avatares/5/E/3/6/4/7/Fraudier/120x120_493.jpg',
      site: 'http://www.fraudierdrogbaa.com',
      gender: 'm',
      country: 'VG',
      message: 'El soldado de fermas',
      migrated: [],
      subscribedToDefaultChannels: false,
    },
    subscribers: 0,
    stories: 1120,
    latestStory: '2018-02-06T04:19:50.000Z',
    state: {
      subscription: 'unsubscribed',
    },
  },
  owner: {
    id: 'ixoiq',
    username: 'Fraudier',
    type: 'user:summary',
    firstname: 'Fraudier',
    lastname: 'ㅤ',
    avatar:
      'https://a12.t26.net/taringa/avatares/5/E/3/6/4/7/Fraudier/120x120_493.jpg',
    gender: 'm',
    country: 'VG',
  },
  upvotes: 2905,
  downvotes: 0,
  shares: 1,
  comments: 78,
  state: {},
  brandSafe: false,
  listViewSafe: false,
}

export const text = {
  id: 'zcvzz',
  created: 1513920502005,
  edited: null,
  title: '',
  nsfw: false,
  classic: {
    tid: 'cHDmR',
    type: 'shout',
  },
  tags: ['text', 'classic'],
  slug:
    'el-de-kapanga-echo-a-un-pibe-del-recital-por-tener-una-remera-con-la-b_zcvzz',
  type: 'story:summary',
  summary: {
    link: null,
    images: {
      slice: [],
      amount: 0,
    },
    excerpt:
      'El de Kapanga echó a un pibe del recital por tener una remera con la bandera inglesa, temiendo que pudiera incitar a su público a...',
    textLength: 167,
  },
  channel: {
    type: 'channel:summary',
    id: 'fswgi',
    created: '2016-08-17T22:02:01.000Z',
    name: 'user-Fixie-',
    title: 'user-Fixie-',
    channelType: 'user_feed',
    thumbnail: '',
    background: '',
    description: '',
    category: 'unknown/unknown',
    owner: {
      id: 'fswgh',
      created: '2016-08-17T22:02:01.000Z',
      type: 'user',
      tid: 24379422,
      username: 'Fixie-',
      firstname: 'Idiota',
      lastname: '.',
      birthday: '1998-4-4',
      avatar:
        'https://a21.t26.net/taringa/avatares/9/E/2/C/1/5/Fixie-/120x120_A50.jpg',
      site: 'http://untitled-pibe.tumblr.com/',
      gender: 'm',
      country: 'IT',
      message: 'zisis a bath tub',
      migrated: [],
      subscribedToDefaultChannels: false,
    },
    subscribers: 0,
    stories: 5978,
    latestStory: '2018-02-07T22:04:49.000Z',
    state: {
      subscription: 'unsubscribed',
    },
  },
  owner: {
    id: 'fswgh',
    username: 'Fixie-',
    type: 'user:summary',
    firstname: 'Idiota',
    lastname: '.',
    avatar:
      'https://a21.t26.net/taringa/avatares/9/E/2/C/1/5/Fixie-/120x120_A50.jpg',
    gender: 'm',
    country: 'IT',
  },
  upvotes: 41,
  downvotes: 0,
  shares: 0,
  comments: 0,
  state: {},
  brandSafe: false,
  listViewSafe: false,
}

export const article = {
  id: 'zcwgr',
  created: 1513920502005,
  edited: '2018-02-07T21:39:24.723Z',
  title: 'consejos para sobrevivir al ataque de un animal salvaje',
  nsfw: false,
  classic: {
    tid: 20125784,
    type: 'post',
    images: {
      '1:1': {
        type: 'image',
        url: 'https://t3.kn3.net/taringa/4/B/B/5/4/A/arcangelg80/01A.jpg',
        width: 526,
        height: 526,
        format: 'jpeg',
      },
      '4:3': {
        type: 'image',
        url: 'https://t1.kn3.net/taringa/F/0/C/8/7/C/arcangelg80/42E.jpg',
        width: 526,
        height: 395,
        format: 'jpeg',
      },
    },
  },
  tags: ['article', 'classic'],
  slug: 'consejos-para-sobrevivir-al-ataque-de-un-animal-salvaje_zcwgr',
  type: 'story:summary',
  summary: {
    link: null,
    images: {
      slice: [
        {
          type: 'image',
          url: 'https://t1.kn3.net/taringa/F/0/C/8/7/C/arcangelg80/42E.jpg',
          width: 526,
          height: 395,
          format: 'jpeg',
        },
        {
          type: 'image',
          url: 'https://k30.kn3.net/taringa/1/C/B/6/D/0/MaestroRoshiII/01B.gif',
          width: 650,
          height: 200,
          format: 'gif',
        },
        {
          type: 'image',
          url: 'https://k50.kn3.net/taringa/6/5/6/9/3/5/868.gif',
          width: 435,
          height: 250,
          format: 'gif',
        },
      ],
      amount: 10,
    },
    excerpt:
      'https://kn3.net/MaestroRoshiII/30-1-C-B-6-D-0-01B-GIF.html\nSi bien es cierto que es muy raro ser atacado por animales salvajes, por otro ...',
    textLength: 3926,
  },
  channel: {
    type: 'channel:summary',
    id: '22e',
    created: '2016-08-05T17:55:08.000Z',
    name: 'hazlo_tu_mismo',
    title: 'Hazlo tú mismo',
    channelType: 'public',
    thumbnail: 'https://k60.kn3.net/taringa/E/5/8/7/7/B/FB6.png',
    background: 'https://k60.kn3.net/taringa/F/1/F/6/F/8/E8D.png',
    description: 'Hazlo tú mismo',
    category: 'diy/tutorials',
    owner: {
      id: '17t',
      created: '2016-07-29T16:57:34.000Z',
      type: 'user',
      tid: 2987,
      username: 'Taringa',
      firstname: 'Taringa!',
      lastname: 'Inteligencia Colectiva',
      birthday: '1994-4-14',
      avatar:
        'https://a11.t26.net/taringa/avatares/3/D/6/2/3/8/Taringa/120x120_943.jpg',
      site: 'http://www.taringa.net',
      gender: 'm',
      country: 'AR',
      message: 'Inteligencia Colectiva',
      migrated: ['followers', 'classicCategories'],
      subscribedToDefaultChannels: true,
    },
    subscribers: 319050,
    stories: 45398,
    latestStory: '2018-02-07T22:35:28.568Z',
    state: {
      subscription: 'unsubscribed',
    },
  },
  owner: {
    id: 'a7716',
    username: 'arcangelg80',
    type: 'user:summary',
    firstname: 'angel',
    lastname: 'gonzalez',
    avatar:
      'https://a21.t26.net/taringa/avatares/A/4/6/B/2/0/arcangelg80/120x120_CDD.jpg',
    gender: 'm',
    country: 'VE',
  },
  upvotes: 9,
  downvotes: 0,
  shares: 0,
  comments: 14,
  state: {},
  brandSafe: true,
  listViewSafe: false,
}

export const link = {
  id: 'zcxyl',
  created: 1513920502005,
  edited: null,
  title: 'Godoy Cruz-Lanús, Superliga: juegan en Mendoza un partido pendiente',
  nsfw: false,
  tags: ['link', 'brainiac'],
  slug:
    'godoy-cruz-lanus-superliga-juegan-en-mendoza-un-partido-pendiente_zcxyl',
  type: 'story:summary',
  summary: {
    link: {
      type: 'link',
      url: 'https://www.lanacion.com.ar/2107299-godoy-cruz-lanus-superliga',
      resourceType: 'html',
      originalUrl:
        'http://www.lanacion.com.ar/2107299-godoy-cruz-lanus-superliga?utm_source=taringa&utm_campaign=rss',
      provider: {
        type: 'provider',
        url: 'https://www.lanacion.com.ar',
        name: 'Lanacion',
        display: 'www.lanacion.com.ar',
      },
      favicon: 'https://k50.kn3.net/taringa/F/4/C/5/E/8/3D0.png',
      title:
        'Godoy Cruz-Lanús, Superliga: juegan en Mendoza un partido pendiente',
      description:
        "Godoy Cruz y Lanús juegan hoy en Mendoza el partido que tienen pendiente de la décima fecha de la Superliga del fútbol argentino, postergado por la participación del 'Granate´ en la final de la Copa Libertadores 2017.",
      images: [
        {
          type: 'image',
          url: 'https://k50.kn3.net/taringa/9/F/2/5/E/D/EB2.jpg',
          width: 900,
          height: 599,
          format: 'jpeg',
        },
      ],
      media: null,
      authors: [],
    },
    images: {
      slice: [],
      amount: 0,
    },
    excerpt: '',
    textLength: 0,
  },
  channel: {
    type: 'channel:summary',
    id: '1jq',
    created: '2016-08-04T21:06:31.000Z',
    name: 'deportes',
    title: 'Deportes',
    channelType: 'public',
    thumbnail: 'https://k60.kn3.net/taringa/9/E/E/B/3/0/589.png',
    background: 'https://k60.kn3.net/taringa/C/7/E/1/C/A/E4C.jpg',
    description: 'Deportes',
    category: 'sports/other',
    owner: {
      id: '17t',
      created: '2016-07-29T16:57:34.000Z',
      type: 'user',
      tid: 2987,
      username: 'Taringa',
      firstname: 'Taringa!',
      lastname: 'Inteligencia Colectiva',
      birthday: '1994-4-14',
      avatar:
        'https://a11.t26.net/taringa/avatares/3/D/6/2/3/8/Taringa/120x120_943.jpg',
      site: 'http://www.taringa.net',
      gender: 'm',
      country: 'AR',
      message: 'Inteligencia Colectiva',
      migrated: ['followers', 'classicCategories'],
      subscribedToDefaultChannels: true,
    },
    subscribers: 317511,
    stories: 275050,
    latestStory: '2018-02-07T23:25:13.000Z',
    state: {
      subscription: 'unsubscribed',
    },
  },
  owner: {
    id: '1kphe',
    username: 'lanacion',
    type: 'user:summary',
    firstname: 'La Nación',
    lastname: 'RSS',
    avatar:
      'https://a21.t26.net/taringa/avatares/D/0/8/9/1/4/lanacion/120x120_96C.jpg',
    gender: 'm',
    country: 'AR',
  },
  upvotes: 0,
  downvotes: 0,
  shares: 0,
  comments: 0,
  state: {},
  brandSafe: false,
  listViewSafe: false,
}
