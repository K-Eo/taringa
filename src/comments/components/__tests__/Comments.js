import React from 'react'
import { mount } from 'enzyme'
import renderer from 'react-test-renderer'
import { MemoryRouter } from 'react-router-dom'

import { Comments } from '../Comments'
import { comment, comments } from '../comment.data'

describe('Comments', () => {
  beforeEach(() => {
    Date.now = jest.fn(() => 1513924592705)
  })

  it('exists', () => {
    expect(Comments).toBeDefined()
  })

  it('renders intial fetching', () => {
    const wrapper = shallow(<Comments />)
    expect(wrapper).toMatchSnapshot()
  })

  it('renders empty', () => {
    const wrapper = shallow(<Comments totalCount={0} status="success" />)
    expect(wrapper).toMatchSnapshot()
  })

  it('renders listing one', () => {
    const wrapper = shallow(
      <Comments totalCount={1} status="success" comments={[comment]} />
    )

    expect(wrapper).toMatchSnapshot()
  })

  it('renders listing two', () => {
    const wrapper = shallow(
      <Comments totalCount={2} status="success" comments={comments} />
    )

    expect(wrapper).toMatchSnapshot()
  })

  it('renders listing two and load more', () => {
    const wrapper = shallow(
      <Comments totalCount={3} status="success" comments={comments} />
    )

    expect(wrapper).toMatchSnapshot()
  })

  it('renders loading more', () => {
    const wrapper = shallow(
      <Comments totalCount={1} status="fetching" comments={[comment]} />
    )

    expect(wrapper).toMatchSnapshot()
  })

  it('renders with initial error', () => {
    const wrapper = shallow(<Comments totalCount={1} status="failure" />)

    expect(wrapper).toMatchSnapshot()
  })

  it('renders with error', () => {
    const wrapper = shallow(
      <Comments totalCount={1} status="failure" comments={[comment]} />
    )

    expect(wrapper).toMatchSnapshot()
  })

  it('handles load more', () => {
    const mock = jest.fn()

    const wrapper = shallow(
      <Comments
        totalCount={3}
        loadMore={mock}
        status="success"
        comments={comments}
      />
    )

    wrapper.find('button').simulate('click')

    expect(mock).toBeCalled()
  })

  it('handles load more when has failure', () => {
    const mock = jest.fn()

    const wrapper = shallow(
      <Comments
        totalCount={3}
        loadMore={mock}
        status="failure"
        comments={comments}
      />
    )

    wrapper.find('button').simulate('click')

    expect(mock).toBeCalled()
  })
})
