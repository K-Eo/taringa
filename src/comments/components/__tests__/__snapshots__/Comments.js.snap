// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Comments renders empty 1`] = `
<div
  className="Comments mt-4"
>
  <div
    className="card"
  >
    <div
      className="card-body"
    >
      <h6
        className="card-title mb-0"
      >
        No hay comentarios
      </h6>
    </div>
  </div>
</div>
`;

exports[`Comments renders idle 1`] = `null`;

exports[`Comments renders listing 1`] = `
<div
  className="Comments mt-4"
>
  <div
    className="card"
  >
    <div
      className="card-body"
    >
      <h6
        className="card-title mb-0"
      >
        1 comentario
      </h6>
    </div>
    <div
      className="card-body"
    >
      <div
        className="Comment mb-4 d-flex"
      >
        <div
          className="Comment-avatar mr-3"
        >
          <img
            alt="Foobar"
            className="rounded Comment-avatar"
            src="https://a18.t26.net/avatares/2/3/4/5/120x120_nr_23454008.jpg?888730"
          />
        </div>
        <div
          className="Comment-content"
        >
          <h6
            className="Comment-author"
          >
            <a
              href="/"
            >
              Foobar
            </a>
            <time
              className="Comment-meta-created"
              dateTime="2017-12-22T05:28:22.005Z"
              title="2017-12-22 05:28"
            >
              hace 1 hora
            </time>
          </h6>
          <div
            className="Comment-body"
            dangerouslySetInnerHTML={
              Object {
                "__html": "I am a text comment",
              }
            }
          />
        </div>
      </div>
    </div>
    <div
      className="text-center"
    >
      <p
        className="text-secondary"
      >
        No hay más comentarios
      </p>
    </div>
  </div>
</div>
`;

exports[`Comments renders listing with error 1`] = `
<div
  className="Comments mt-4"
>
  <div
    className="card"
  >
    <div
      className="card-body"
    >
      <h6
        className="card-title mb-0"
      >
        1 comentario
      </h6>
    </div>
    <div
      className="card-body"
    >
      <div
        className="Comment mb-4 d-flex"
      >
        <div
          className="Comment-avatar mr-3"
        >
          <img
            alt="Foobar"
            className="rounded Comment-avatar"
            src="https://a18.t26.net/avatares/2/3/4/5/120x120_nr_23454008.jpg?888730"
          />
        </div>
        <div
          className="Comment-content"
        >
          <h6
            className="Comment-author"
          >
            <a
              href="/"
            >
              Foobar
            </a>
            <time
              className="Comment-meta-created"
              dateTime="2017-12-22T05:28:22.005Z"
              title="2017-12-22 05:28"
            >
              hace 1 hora
            </time>
          </h6>
          <div
            className="Comment-body"
            dangerouslySetInnerHTML={
              Object {
                "__html": "I am a text comment",
              }
            }
          />
        </div>
      </div>
    </div>
    <div
      className="text-center"
    >
      <p
        className="text-danger"
      >
        ¡Ratas! Algo salío mal. Refresca la página o haz clic
         
        <a
          href="/retry"
          onClick={[Function]}
        >
          aquí
        </a>
         
        para volver a intentar.
      </p>
    </div>
  </div>
</div>
`;

exports[`Comments renders listing with loading more 1`] = `
<div
  className="Comments mt-4"
>
  <div
    className="card"
  >
    <div
      className="card-body"
    >
      <h6
        className="card-title mb-0"
      >
        2 comentarios
      </h6>
    </div>
    <div
      className="card-body"
    >
      <div
        className="Comment mb-4 d-flex"
      >
        <div
          className="Comment-avatar mr-3"
        >
          <img
            alt="Foobar"
            className="rounded Comment-avatar"
            src="https://a18.t26.net/avatares/2/3/4/5/120x120_nr_23454008.jpg?888730"
          />
        </div>
        <div
          className="Comment-content"
        >
          <h6
            className="Comment-author"
          >
            <a
              href="/"
            >
              Foobar
            </a>
            <time
              className="Comment-meta-created"
              dateTime="2017-12-22T05:28:22.005Z"
              title="2017-12-22 05:28"
            >
              hace 1 hora
            </time>
          </h6>
          <div
            className="Comment-body"
            dangerouslySetInnerHTML={
              Object {
                "__html": "I am a text comment",
              }
            }
          />
        </div>
      </div>
    </div>
    <div
      className="card-body"
    >
      <div
        className="text-center"
      >
        <i
          className="fa fa-circle-notch fa-spin"
        />
      </div>
    </div>
  </div>
</div>
`;

exports[`Comments renders listing with more than one 1`] = `
<div
  className="Comments mt-4"
>
  <div
    className="card"
  >
    <div
      className="card-body"
    >
      <h6
        className="card-title mb-0"
      >
        2 comentarios
      </h6>
    </div>
    <div
      className="card-body"
    >
      <div
        className="Comment mb-4 d-flex"
      >
        <div
          className="Comment-avatar mr-3"
        >
          <img
            alt="Foobar"
            className="rounded Comment-avatar"
            src="https://a18.t26.net/avatares/2/3/4/5/120x120_nr_23454008.jpg?888730"
          />
        </div>
        <div
          className="Comment-content"
        >
          <h6
            className="Comment-author"
          >
            <a
              href="/"
            >
              Foobar
            </a>
            <time
              className="Comment-meta-created"
              dateTime="2017-12-22T05:28:22.005Z"
              title="2017-12-22 05:28"
            >
              hace 1 hora
            </time>
          </h6>
          <div
            className="Comment-body"
            dangerouslySetInnerHTML={
              Object {
                "__html": "I am a text comment",
              }
            }
          />
        </div>
      </div>
      <div
        className="Comment mb-4 d-flex"
      >
        <div
          className="Comment-avatar mr-3"
        >
          <img
            alt="Foobar"
            className="rounded Comment-avatar"
            src="https://a18.t26.net/avatares/2/3/4/5/120x120_nr_23454008.jpg?888730"
          />
        </div>
        <div
          className="Comment-content"
        >
          <h6
            className="Comment-author"
          >
            <a
              href="/"
            >
              Foobar
            </a>
            <time
              className="Comment-meta-created"
              dateTime="2017-12-22T05:28:22.005Z"
              title="2017-12-22 05:28"
            >
              hace 1 hora
            </time>
          </h6>
          <div
            className="Comment-body"
            dangerouslySetInnerHTML={
              Object {
                "__html": "I am a text comment",
              }
            }
          />
        </div>
      </div>
    </div>
    <div
      className="text-center"
    >
      <p
        className="text-secondary"
      >
        No hay más comentarios
      </p>
    </div>
  </div>
</div>
`;
